src/App.js

```diff
@@ -10,8 +10,10 @@ import { setCompleted, setNotCompleted, setLocalGroups } from './db/store';
 function App(props) {
   const [groups, setGroups] = useState([])
   const [filter, setFilter] = useState({selectedSort: '', searchGroup: ''})
+  const [modalGroupVisible, setModalGroupVisible] = useState(false)
   const addGroup = (newGroup) => {
     setGroups([...groups, newGroup])
+    setModalGroupVisible(false)
   }
   const removeGroup = (group) => {
     setGroups(groups.filter(i => i.id !== group.id))
```

Comment:
1. Parentheses are always around the arguments of arrow functions:
```suggestion
    setGroups(groups.filter((i) => i.id !== group.id))
```